### ISSUE TITLE
Wrong editing style after quitting spray (speed-reading)

### ISSUE THREAD

**Issue by jakuzure** [syl20bnr/spacemacs#5205]
#### Description
I'm using the emacs editing style. When I `start-spray` via M-m a R the color code changes to green (insert mode) while it's in spray mode. After it either finished or I quit, the color code changes to orange aka vim mode.

#### Reproduction guide
- Start Emacs
- start-spray while using emacs editing style
- wait until it's finished or quit by pressing q

*Observed behaviour:*
Editing style changed to vim,

*Expected behaviour:*
Editing style should still be emacs style.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.11
- Spacemacs branch: master (rev. 6689699)
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp racket lua git markdown org latex
                 (shell :variables shell-default-height 30 shell-default-position 'bottom)
                 spell-checking syntax-checking version-control speed-reading colors)
```

**Comment by StreakyCobra** [comment]
@AlejandroCatalina Following #5111, do you want to continue with https://github.com/syl20bnr/spacemacs/blob/83556719cbe2849fb018a6bbc85fe4a7876cc207/layers/speed-reading/packages.el#L34 :smiley: ?

**Comment by jakuzure** [comment]
I tried fixing this myself here: https://github.com/syl20bnr/spacemacs/pull/5207

**Comment by AlejandroCatalina** [comment]
@StreakyCobra sure! I'll go for it if it is not solved in the PR.

Sent from my Cyanogen phone

On Feb 24, 2016 02:10, 蛇崩乃音 <notifications@github.com> wrote:

I tried fixing this myself here: #5207

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
